public/app/plugins/datasource/azuremonitor/components/ConfigEditor/MonitorConfig.tsx

```diff
@@ -48,6 +48,11 @@ export const MonitorConfig = (props: Props) => {
   const onSubscriptionChange = (subscriptionId?: string) =>
     updateOptions((options) => ({ ...options, jsonData: { ...options.jsonData, subscriptionId } }));
 
+  if (!options.jsonData.authType) {
```

Comment:
I wonder if this would make more sense being in the `AzureCredentialsForm` file as a `useEffectOnce` so that it's closer to the component which sets the auth type (and also the default authentication type value). Wdyt?